include/bsp/iotc_bsp_crypto.h

```diff
@@ -29,115 +29,96 @@ extern "C" {
 
 /**
  * @file  iotc_bsp_crypto.h
- * @brief Encode and sign JWT credentials.
- *
- * These functions implement a cryptography library to: generate and sign JWTs
- * via ECC keys and SHA256; or format signed JWTs as URL-safe base64 strings.
- * Reference implementations are in the
- * <a href="~/src/bsp/crypto"><code>src/bsp/crypto</code></a> directory.
- *
- * The BSP uses the private key data or slot number provided to the
- * <code>iotc_connect()</code> function. The BSP doesn't format the JWT itself;
- * the BSP only signs JWTs. The <a href="../../api/html/index.html">Device SDK
- * API</a> formats the original JWT.
- *
- * All cryptography functions issue an <code>iotc_bsp_crypto_state_e</code>
- * status message.
+ * @brief Implements a cryptography library for creating JSON Web Tokens.
+ * 
+ * @details The TLS library:
+ *     - Generates JWT credentials.
+ *     - Signs JWTs with Elliptic Curve cryptography and SHA256.
+ *     - Encodes signed JWTs as URL-safe base64 strings.
+ *
+ * The SDK has two <a href="../../bsp/html/index.html">turn-key TLS libraries
+ * for POSIX platforms: <a href="../../../src/bsp/crypto/mbedtls">mbedTLS</a>
+ * and <a href="../../../src/bsp/crypto/wolfssl">wolfSSL</a>. You can modify the
+ * functions in this file to create a new TLS library.
```

Comment:
...to create a new cryptography library.

Reply:
Done